internal/serve/httphandler/disbursement_handler_test.go

```diff
@@ -854,6 +854,17 @@ func Test_DisbursementHandler_PostDisbursementInstructions(t *testing.T) {
 			expectedStatus:  http.StatusOK,
 			expectedMessage: "File uploaded successfully",
 		},
+		{
+			name:           "valid input",
+			disbursementID: draftDisbursement.ID,
+			csvRecords: [][]string{
+				{"phone", "id", "amount", "verification"},
+				{"+380445555555", "123456789", "100.5", "1990-01-01"},
+				{"+380445555555", "123456789", "100.5", "1990-01-01"},
+			},
+			expectedStatus:  http.StatusOK,
+			expectedMessage: "File uploaded successfully",
+		},
```

Comment:
Is the idea here to validate that you can have multiple payments to the same receiver? If so, can you update the test name to make it clear?

Reply:
No Sorry, I was testing something else there and forgot to remove this test. I'm removing it now.